runners/core-construction-java/src/main/java/org/apache/beam/runners/core/construction/CombineTranslation.java

```diff
@@ -84,6 +91,16 @@ public FunctionSpec translate(
       }
     }
 
+    @Override
+    public @Nullable Row toConfigRow(PTransform<?, ?> pTransform) {
```

Comment:
I don't think these need  a separate interface but good point regarding providing default implementations. Made "toConfigRow" and "fromConfigRow" default methods of the interface so that sub-classes can choose to not implement them.